README.md

```diff
@@ -59,7 +59,7 @@ ember install @bagaar/ember-breadcrumbs
 </BreadcrumbsItem>
 ```
 
-> **NOTE:** The `<BreadcrumbsItem />` component is responsible for rendering the provided `{{link-to}}` component into all instances of the `<Breadcrumbs />` component using Ember's `{{-in-element}}` component.
+> **NOTE:** The `<BreadcrumbsItem />` component is responsible for rendering the provided `{{link-to}}` component into all instances of the `<Breadcrumbs />` component using Ember's `{{in-element}}` helper.
```

Comment:
It's not a helper, right? Since it takes block level content? Not sure what it is though.

Reply:
It is actually, using `component` before was incorrect. `let` and `each` are also template helpers that accept content. Ref => https://github.com/emberjs/ember.js/issues/17947

Reply:
I don't think it is, it's a low level Glimmer VM api exposed to Ember. I just think some people call it a helper because it looks like one. Similar to how the action modifier was called a helper before. I'm guessing the same is true for most of the other built in template helpers.

Anyway, if they officially call it a helper, fine by me. :slightly_smiling_face: :+1: